config/jobs/kubernetes-sigs/sig-storage-local-static-provisioner/sig-storage-local-static-provisioner-trusted.yaml

```diff
@@ -0,0 +1,45 @@
+postsubmits:
+  kubernetes-sigs/sig-storage-local-static-provisioner:
+  # A postsubmit job to build canary build on master.
+  - name: post-sig-storage-local-static-provisioner-build-canary
+    branches:
+    - master
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/sig-storage-local-static-provisioner
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190108-eadd44c98-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - e2e
+        - release
+        env:
+        - name: PROVIDER
```

Comment:
don't need these 2 envs?

Reply:
You mean why I add these 2 envs? It runs e2e before building image, I want to make it clearer.

Reply:
thanks for the clairifcation, this lgtm
/approve

Reply:
e2e target is removed